src/components/PostList/PostList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import Post from '../Post/Post';
+
+function PostList({ posts }) {
+  return (
+   <div>
+     { 
+       posts.map(post => 
+         <Post {...post} key={post.id} />)
+     }
+   </div>
+  )
+}
+
+PostList.propTypes = {
+  posts: propTypes.arrayOf(
+    propTypes.shape({
+      id: propTypes.number.isRequired,
+    }).isRequired,
+  ).isRequired,
+};
+
+PostList.defaultProps = {
```

Comment:
```suggestion
```
You don't need to set the default props because you've described your `posts` array as `required`